### DISCUSSION TITLE
Création de compétences

### DISCUSSION THREAD

**Discussion by Moigaston** [chamilo/chamilo-lms#5796] [General]
je viens de visionner la vidéo-tuto  sur les compétences de "Monsieur le président" :) pour comprendre comment créer des compétences. J'ai trois questions:
1. Comment supprimer une compétence dans la rubrique administration?
2. D'après ce que je constate, seul l’administrateur peut créer des compétences - Est-ce le cas?
3. Est-ce que le concept de certificat est lié à celui de compétence, à savoir qu'il faut obtenir un certain nombre de certificats pour acquérir une compétence?
Merci d'avance pour toute réponse!

**Comment by NicoDucou** [comment]
Bonjour @Moigaston 

> je viens de visionner la vidéo-tuto sur les compétences de "Monsieur le président" :) pour comprendre comment créer des compétences. J'ai trois questions:
> 
>     1. Comment supprimer une compétence dans la rubrique administration?
Ce n'est pas possible de supprimer une compétence, on peut juste les inactivées. C'est un choix qui a été fait pour ne pas perdre d'information qui y serait lié car les openbadges sont sensés avoir un lien de vérification qui existe toujours. C'est quelque chose qui évoluera peut-être dans les années à venir, mais pour le moment ça fonctionne comme ça et pour réellement supprimer un élément il faut le faire dans la base de données et donc savoir ce que l'on fait exactement.
> 
>     2. D'après ce que je constate, seul l’administrateur peut créer des compétences - Est-ce le cas?
Oui c'est bien cela, seulement depuis l'interface d'administration.
> 
>     3. Est-ce que le concept de certificat est lié à celui de compétence, à savoir qu'il faut obtenir un certain nombre de certificats pour acquérir une compétence?
Non c'est totalement indépendant. Les certificats s'obtiennent en fonction de résultat dans les cahiers de notes des cours.
Les compétences peuvent être assigné par un professeur ou un supérieur d'apprenant mais également en fonction de résultat dans le cahier de note, mais indépendement des certificats.

Cordialement

**Comment by chatonito** [comment]
Je ne parle pas français, répond avec Google Traducteur

1.- Vous ne pouvez pas supprimer les compétences dans l'administration, uniquement depuis la base de données SQL, je suggère de ne pas la déplacer, cela peut tout endommager, si vous souhaitez supprimer une compétence d'un utilisateur, vous devrez la supprimer complètement (toute). ses utilisateurs sont supprimés).

2.- dans configuration.php, vous pouvez donner à l'enseignant la possibilité d'attribuer des compétences (aptitudes), 

3.- le certificat est facultatif, dans la configuration des qualifications, vous pouvez attribuer ou non un certificat à l'aptitude ou à la compétence, ou vous pouvez créer des aptitudes et des compétences sans certificat

J'espère avoir aidé
atte
Dr. Agustín Rico Guzmán
webmaster Éducation Médicale CAR Mexique
www.profemedico.com.mx

**Comment by Moigaston** [comment]
Merci beaucoup NicoDucou pour votre réponse rapide et précise !
Et merci aussi à vous pour les renseignements chatonito. Bon à savoir!
